peripherals/rcc/rcc_v3.yaml

```diff
@@ -0,0 +1,124 @@
+# Applicable at least to H7
+
+# Fresh start - even fields from rcc_common are quite different
+
+RCC:
+  CR:
+    HSIDIVF:
+      _read:
+        NotPropagated: [0, "New HSIDIV ratio has not yet propagated to hsi_ck"]
+        Propagated: [1, "HSIDIV ratio has propagated to hsi_ck"]
+    HSEBYP:
+      NotBypassed: [0, "HSE crystal oscillator not bypassed"]
+      Bypassed: [1, "HSE crystal oscillator bypassed with external clock"]
+    HSIDIV:
+      Div1: [0, "No division"]
+      Div2: [1, "Division by 2"]
+      Div4: [2, "Division by 4"]
+      Div8: [3, "Division by 8"]
+    "*RDY":
+      _read:
+        NotReady: [0, "Clock not ready"]
+        Ready: [1, "Clock ready"]
+    "*ON":
+      "Off": [0, "Clock Off"]
+      "On": [1, "Clock On"]
+  CRRCR:
+    _read:
+      HSI48CAL: [0, 0x3FF]
+  CFGR:
+    MCO2:
+      SYS: [0, "System clock selected for micro-controller clock output"]
```

Comment:
We use `SYSCLK` everywhere else for this one.

Reply:
Sure - Done